src/app/components/dynamic-hub/dynamic-hub.component.html

```diff
@@ -17,7 +17,19 @@
     <div class="dynamic-hub__content">
         <div class="dashboards">
             <div class="dashboard" *ngFor="let data of cards | keyvalue">
-                <h2>{{data.key | titlecase | translate}} {{'dashboards' | translate}}</h2>
+                <h2>
+                    <span class="dashboard-title">{{data.key | titlecase | translate}} {{'dashboards' | translate}}</span>
```

Comment:
we could use the key with variable substitution

Reply:
I prefer to keep it as it is for now